pupil-spa/src/app/accessibility-statement/accessibility-statement.component.html

```diff
@@ -151,21 +165,32 @@ <h3 class="heading-medium">Enforcement procedure</h3>
         please <a href="https://www.equalityadvisoryservice.com/">contact the Equality Advisory and Support Service (EASS).</a>
       </p>
 
-      <p>The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications)
-        (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’).
+      <p>The Equality and Human Rights Commission
+        (EHRC) is responsible for enforcing the <a
+        class="govuk-link" target="_blank" rel="noopener noreferrer"
+        href="https://www.legislation.gov.uk/uksi/2018/952/contents/made">
+        Public Sector Bodies (Websites and Mobile Applications) (No. 2)
+        Accessibility Regulations 2018 (the ‘accessibility
+        regulations’)</a>.
       </p>
 
       <h3 class="heading-medium">How we tested this website</h3>
 
-      <p>This website was and is currently being tested for compliance with the Web Content Accessibility Guidelines V2.1 level A and level AA,
-        and these tests have been carried out internally.
+      <p>We have tested this website for compliance
+        with the <a href="https://www.w3.org/TR/WCAG21/" class="govuk-link"
+      target="_blank" rel="noopener noreferrer">
+      Web Content Accessibility Guidelines V2.1</a> level A and level
+      AA, and these tests have been carried out internally.
       </p>
 
       <h3 class="heading-medium">What we’re doing to improve accessibility</h3>
 
-      <p>Ahead of the 2021/22 MTC cycle we will audit our content again and update the service according to report findings.</p>
+      <p class="govuk-body">We audit our content annually and update the service according to report findings.</p>
 
-      <p>This statement was prepared based on an independent accessibility audit carried out in July 2019. It was last updated on 19 April 2021.</p>
+      <p>This statement was prepared based on an
+        independent accessibility audit of the full MTC service carried out
+        in July 2019. It was last updated on 05 April 2023 following an
```

Comment:
There is no need to zero-pad dates. As per GDS style guide it should be '5 April 2023'
https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style

Reply:
this needs raising internally, copied verbatim